ManualText.txt

```diff
@@ -910,7 +881,7 @@ Work orders for CTS usually only fall under the category of major network emerge
 Dispatchers may enter CTS work requests for issues involving campus phones and technology if instructed by the officer (as with Physical Plant work orders), which can be done by emailing CTS@gordon.edu.  Always use the dispatch email function for these emails so that they are sent from the police@gordon.edu email address rather than your personal email address.
 
 8.2	Escort Log (App. Escort Log)
-In the event of an escort, you need to log the information in Report Exec and also on a separate escort log, containing other necessary information used for quality control purposes. The escort log can be found in a binder to the right of the main dispatch console. In Report Exec, record the name of the caller, the time, and "Assistance: Escort" as the call type. In the escort log, put the time that the call came in, the name of the person requesting the escort, where he or she is going (from and to), the officer responding, his or her arrival time, and his or her clear time.
+In the event of an escort, you need to log the information in Report Exec. In Report Exec use "Assistance: Escort" as the call type. Make sure to include the time that the call came in, the name of the person requesting the escort, where he or she is going (from and to), the officer responding, his or her arrival time, and his or her clear time.
```

Comment:
We can probably move this to another section where it is not out of place now that there is not going to be a separate log.